distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/VideoTest.java

```diff
@@ -0,0 +1,229 @@
+import static com.jayway.restassured.RestAssured.when;
+import static ddf.common.test.WaitCondition.expect;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.DatagramSocket;
+import java.net.SocketException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.io.IOUtils;
+import org.codice.alliance.distribution.sdk.video.stream.mpegts.MpegTsUdpClient;
+import org.codice.alliance.video.stream.mpegts.UdpStreamMonitor;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+
+import com.jayway.restassured.response.ValidatableResponse;
+
+import ddf.common.test.BeforeExam;
+import ddf.test.itests.AbstractIntegrationTest;
+
+/**
+ * Tests Alliance video capabilities.
+ */
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class VideoTest extends AbstractIntegrationTest {
+    private static final String[] REQUIRED_APPS = {"catalog-app", "solr-app", "video-app"};
+
+    private static final String METACARD_COUNT_XPATH = "count(/metacards/metacard)";
+
+    private static final String METACARD_ID_XMLPATH = "metacards.metacard.@gml:id";
+
+    private static final String LOCALHOST = "127.0.0.1";
+
+    private static final String DAYFLIGHT = "dayflight.mpg";
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    private DynamicPort udpPort;
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
+        basePort = getBasePort();
+        udpPort = new DynamicPort(6);
+
+        getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+        getServiceManager().waitForAllBundles();
+        getCatalogBundle().waitForCatalogProvider();
+    }
+
+    @Override
+    protected Option[] configureDistribution() {
+        return options(karafDistributionConfiguration(maven().groupId(
+                "org.codice.alliance.distribution")
+                .artifactId("alliance")
+                .type("zip")
+                .versionAsInProject()
+                .getURL(), "alliance", KARAF_VERSION).unpackDirectory(new File("target/exam"))
+                .useDeployFolder(false));
+    }
+
+    @Override
+    protected Option[] configureCustom() {
+        return options(wrappedBundle(mavenBundle("ddf.test.itests", "test-itests-ddf").classifier(
+                "tests")
+                        .versionAsInProject()).bundleSymbolicName("test-itests-ddf")
+                        .exports("ddf.test.itests.*"),
+                wrappedBundle(mavenBundle("org.codice.alliance.distribution",
+                        "sample-mpegts-streamgenerator").versionAsInProject()),
```

Comment:
Rather than wrap all of these I would recommend creating a features file like we do for the sdk-app in ddf.